test/libsolidity/syntaxTests/literals/invalid_octal_string_no_whitespace.sol

```diff
@@ -0,0 +1,5 @@
+contract C {
+    uint y = 01abc;
```

Comment:
I'd use a denomination here so that the only thing wrong in the test is the thing we're testing for:

```suggestion
    uint y = 01gwei;
```

Reply:
Ok.I will then also rename the file to `invalid_octal_denomination_no_whitespace.sol`.